pathways/en.yml

```diff
@@ -1,6 +1,45 @@
 ---
 en:
   pathways:
+    ai-toolkit:
+      title: "AI toolkit"
+      subtitle: "Engage with a range of artificial intelligence and machine learning platforms!"
+      description: The AI toolkit introduces various artificial intelligence applications and technologies through hands-on projects using different platforms and tools. Students will work with voice recognition, facial recognition, and other AI technologies, gaining a broader understanding of how AI can be applied in different contexts.
```

Comment:
Is this copy going to be primarily read by the kids or the mentors? If it's the kids then perhaps it should be 'you will'. Not sure what the format of the other ones is though so it should be consistent.